backend/src/EventList.WebApi/Common/Exceptions/ApplicationErrorException.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace EventList.WebApi.Common.Exceptions
+{
+    /// <summary>
+    /// Represents application error
+    /// </summary>
+    public class ApplicationErrorException : Exception
```

Comment:
ApplicationError is never used and seems to be generic, do we need it?

Reply:
This exception maps to error code 500. 
Should be used when others do not describe the error enough.
Can be thrown when the request is valid, but business rules are violated.

Reply:
Since it serves a different purpose than ValidationError and it's not generic I'm marking it as resolved